server/src/main/resources/db/changelog/db.changelog-v0.5.0-data.yaml

```diff
@@ -47,4 +47,62 @@ databaseChangeLog:
               - column:
                   name: project_type
                   value: "AN-Projekt"
-            where: id=4
\ No newline at end of file
+            where: id=4
+  - changeSet:
+      id: insert-external-link-jira
+      author: bock
+      context: itzbund
+      changes:
+        - insert:
+            tableName: external_link
+            columns:
+              - column:
+                  name: id
+                  value: 1
+              - column:
+                  name: name
+                  value: Jira
+              - column:
+                  name: description
+                  value: Projektaufgaben verwalten mit Jira
+              - column:
+                  name: url
+                  value: https://jira.swlm.zd.in.bund.de/secure/Dashboard.jspa
+              - column:
+                  name: img
+                  value: /assets/img/mark-gradient-blue-jira.svg
```

Comment:
derzeit haben wir nur wenige Bilder dafür aber ich glaube es macht Sinn, irgendwann die Bilder und co.. auch in der Datenbank zu speichern.

Reply:
Ja, da müssen wir mal schauen... svgs würden sich dafür schon eignen, aber es könnten natürlich auch andere Formate sein.